.cirrus.yml

```diff
@@ -333,6 +333,63 @@ task:
       on_failure:
         <<: *on_failure_ac
 
+    - name: Linux - Debian Bullseye - Meson - Test Extensions
+      env:
+        INSTALL_DIR: ${CIRRUS_WORKING_DIR}/meson-install
+        PATH: ${INSTALL_DIR}/bin:${PATH}
+
+      allow_failures: true
+      depends_on:
+        - Linux - Debian Bullseye - Meson
+
+      configure_script: |
+        su postgres <<-EOF
+          # Disable shadow=compatible-local until
+          # https://github.com/citusdata/pg_cron/issues/214 is fixed
+          export CFLAGS="${CFLAGS} -Wno-shadow=compatible-local"
+          meson setup \
+            --buildtype=debug \
+            --prefix=${INSTALL_DIR} \
+            -Dcassert=true \
+            ${LINUX_MESON_FEATURES} \
+            -DPG_TEST_EXTRA="$PG_TEST_EXTRA" \
+            build
+        EOF
+      build_script: su postgres -c 'ninja -C build -j${BUILD_JOBS}'
+
+      install_script: su postgres -c 'ninja -C build install'
+
+      install_required_packages_for_extensions_script: |
+        apt-get update
+        DEBIAN_FRONTEND=noninteractive apt-get -y install \
+          autoconf \
+          libtool \
+          libgeos-dev \
+          libproj-dev \
+          libprotobuf-c-dev \
+          protobuf-c-compiler
+
+      always:
+        test_pg_qualstats_script: su postgres -c 'python3 src/tools/ci/test_extensions --only pg_qualstats'
+        test_pg_cron_script: su postgres -c 'python3 src/tools/ci/test_extensions --only pg_cron'
+        test_hypopg_script: su postgres -c 'python3 src/tools/ci/test_extensions --only hypopg'
+        test_orafce_script: su postgres -c 'python3 src/tools/ci/test_extensions --only orafce'
+        test_postgis_script: |
+          su postgres <<-EOF
+            export LD_LIBRARY_PATH=${INSTALL_DIR}/lib/x86_64-linux-gnu/:${LD_LIBRARY_PATH}
```

Comment:
Hm. We shouldn't need this, in theory. I think you mentioned on IM that otherwise libpq couldn't be found?  This may indicate a bug leading to the rpath not being set in some circumstances? E.g. pg_qualstats shows
`-Wl,-rpath,'/tmp/cirrus-ci-build/meson-install/lib/x86_64-linux-gnu',--enable-new-dtags` when linking pg_qualstats.so.

I guess this may be needed because postgis doesn't "just" use PGXS but has a lot of its own stuff.

Argh, I found it:
```
# NOTE: setting LDFLAGS explicitly here avoids getting the
# link line polluted by unrelated PostgreSQL switches that
# may be confusing and out-of-control for the builder of
# PostGIS
LDFLAGS =  -lm
```